<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCopyArticleData</class>
 <widget class="QDialog" name="FormCopyArticleData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>143</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_btnBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="m_txtPattern">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Article data copy pattern</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_help" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_cbEscapeCsv">
     <property name="text">
      <string>Escape values for CSV usage</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnBox</sender>
   <signal>accepted()</signal>
   <receiver>FormCopyArticleData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnBox</sender>
   <signal>rejected()</signal>
   <receiver>FormCopyArticleData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
